src/dhcpmon/src/dhcp_device.c

```diff
@@ -33,6 +33,23 @@
 #define DHCP_START_OFFSET (UDP_START_OFFSET + sizeof(struct udphdr))
 /** Start of DHCP Options segment of a captured frame */
 #define DHCP_OPTIONS_HEADER_SIZE 240
+/** Offset of DHCP GIADDR */
+#define DHCP_GIADDR_OFFSET 24
+
+/**
+ * DHCP message types
+ **/
+typedef enum
+{
+   DHCP_MESSAGE_TYPE_DISCOVER       = 1,
+   DHCP_MESSAGE_TYPE_OFFER          = 2,
+   DHCP_MESSAGE_TYPE_REQUEST        = 3,
+   DHCP_MESSAGE_TYPE_DECLINE        = 4,
+   DHCP_MESSAGE_TYPE_ACK            = 5,
+   DHCP_MESSAGE_TYPE_NAK            = 6,
+   DHCP_MESSAGE_TYPE_RELEASE        = 7,
+   DHCP_MESSAGE_TYPE_INFORM         = 8
```

Comment:
Suggest to remove extra spaces for readability. E.g.,

```
   DHCP_MESSAGE_TYPE_DISCOVER = 1,
   DHCP_MESSAGE_TYPE_OFFER    = 2,
   DHCP_MESSAGE_TYPE_REQUEST  = 3,
   DHCP_MESSAGE_TYPE_DECLINE  = 4,
   DHCP_MESSAGE_TYPE_ACK      = 5,
   DHCP_MESSAGE_TYPE_NAK      = 6,
   DHCP_MESSAGE_TYPE_RELEASE  = 7,
   DHCP_MESSAGE_TYPE_INFORM   = 8
```

Reply:
done